src/Sarif.Sdk.Sample/Program.cs

```diff
@@ -0,0 +1,383 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Newtonsoft.Json;
+
+namespace Sarif.Sdk.Sample
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            int result = Parser.Default.ParseArguments<LoadOptions, CreateOptions>(args)
+                .MapResult(
+                    (LoadOptions options) => LoadSarifLogFile(options),
+                    (CreateOptions options) => CreateSarifLogFile(options),
+                    errors => 1);
+
+            return result;
+        }
+
+        /// <summary>
+        /// Loads a SARIF log file from disk and deserializes it into a code object.
+        /// </summary>
+        /// <param name="options"></param>
+        /// <returns></returns>
```

Comment:
If you're not going to populate an XML comment element, just omit it. IMO.

Reply:
Fixed